src/include/log.h

```diff
@@ -368,7 +369,7 @@ struct __wt_log_desc {
 #define WT_LOG_V2_MAJOR 3
 #define WT_LOG_V2_MINOR 0
 #define WT_LOG_V3_MAJOR 3
-#define WT_LOG_V3_MINOR 1
+#define WT_LOG_V3_MINOR 2
```

Comment:
I think you want/need a `WT_LOG_V4_MAJOR/MINOR` settings here too.

Reply:
@sueloverso The `WT_LOG_V3_MAJOR/MINOR` were defined but not used prior to my changes. I can add defines for `WT_LOG_V4_MAJOR/MINOR`, but they would be unused - would that be the right thing to do here?

Reply:
I would prefer it. I think it helps reason about releases and changes when looking in there and any future code changes. You'd need to add them to `dist/s_define.list` also. At some point we can consider cleaning it up or making it a comment or whatever. But for now I suggest adding it to be consistent and clear.

Reply:
Done.